tests/cupy_tests/sorting_tests/test_search.py

```diff
@@ -221,6 +219,21 @@ def test_nonzero(self, xp, dtype):
         return xp.nonzero(array)
 
 
+@testing.parameterize(
+    {'array': numpy.empty(0)},
+    {'array': numpy.empty(1)},
```

Comment:
Should be `numpy.array`. `numpy.empty(0)` and `numpy.empty(1)` result in `ndim=1`.

Reply:
Fixed.